src/NosCore.GameObject/Providers/ItemProvider/Item/WearableInstance.cs

```diff
@@ -146,5 +149,111 @@ public void SetRarityPoint()
                     break;
             }
         }
+
+        public void Sum(ClientSession clientSession, WearableInstance item)
+        {
+            if (!clientSession.HasCurrentMapInstance)
+            {
+                return;
+            }
+
+            if (Upgrade + item.Upgrade > 5)
+            {
+                return;
+            }
+
+            if ((
+                    (Item.EquipmentSlot != EquipmentType.Gloves) ||
```

Comment:
Would be nice not to have this in our gameobject but in the upgrade service

Reply:
Or at least an array of accepted EquipmentTypes instead of a 4 lines if